about.html

```diff
@@ -0,0 +1,94 @@
+                <!-- <div class="divider"></div> -->
+                <li><a href="#">News</a></li>
+                <!-- <div class="divider"></div> -->
+                <li><a href="#">Campaign</a></li>
+                <!-- <div class="divider"></div> -->
+            </ul>
+        </nav>
+      </header>
+      <main>
+        <section class="one">
+          <span class="sub-title">Hello! Music Producers!</span>
+          <h1 class="title" id="title">Tritone Music Producers conference 2023</h1>
+          <small class="description">Lorem ipsum dolor sit amet consectetur adipisicing elit. Tenetur recusandae voluptatibus corporis ipsa temporibus minus incidunt deleniti in earum enim.</small>
+          <small class="contact-us">Please contact us via Email for any further questions about TMC production conference</small>
+          <a class="contact-link" href="#">Tritonemusic.co@enquiries.org</a>
+        </section>
+        <section class="two">
+            <h3>Tritone Music Co Producers Conference Logo</h2>
+            <div class="divider"></div>
+            <p>Lorem ipsum dolor, sit amet consectetur adipisicing elit. Odit labore saepe dolores dolorem laboriosam pariatur eveniet aliquid? Ipsum, architecto possimus.</p>
+            <figure class="logo-img"></figure>
+        </section>
+        <section class="three">
+            <h3>See the past TMC production Conference</h3>
+            <div class="divider"></div>
+            <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Inventore quisquam enim, atque nesciunt doloremque praesentium!</p>
+            <div class="gallery">
+                <div class="image one">
+                    <h3 class="year">2022</h3>
+                    <span class="location">TMC production conference in Accra</span>
+                </div>
+                <div class="image two">
+                    <h3 class="year">2021</h3>
+                    <span class="location">TMC production conference in Kumasi</span>
+                </div>    
+            </div>
+        </section>
+        <footer>
+            <div class="partners-info">
+              <div class="partner-heading">
+                <h2>Partner</h2>
+                <div class="divider"></div>
+              </div>
+              <ul class="partners">
+                <li><a href=""><img src="assets/images/airbnb.png" alt="" class="partner-icon"></a></li>
+                <li><a href=""><img src="assets/images/googlekorea.png" alt="" class="partner-icon"></a></li>
+                <li><a href=""><img src="assets/images/mozilla.png" alt="" class="partner-icon"></a></li>
+                <li><a href=""><img src="assets/images/naver.png" alt="" class="partner-icon"></a></li>
+              </ul>
+            </div>
+            <div class="company-patent">
+              <div class="footer-logo"></div>
+              <div class="copyright">
+                <small>2020 TMC, Some Rights Reserved.</small>
+              </div>
```

Comment:
- [ ] please align the logo and p tage here to be in the center so please align them to the center to match the design 💯🔥and please decrease the size of the logo since it is very big in your design 🔥🙏🏻

- Your design
![image](https://user-images.githubusercontent.com/58553711/212223312-d127f156-60a2-40da-858a-20a97b89071a.png)


- Given
![image](https://user-images.githubusercontent.com/58553711/212223508-f867917c-f5b7-4e2f-9dca-02cc3d39191f.png)